bash/jhail

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source "$(dirname $0)"/setup_env
+
+jupyter nbextension list 2>&1 | grep 'jupyter_spark *enabled'
```

Comment:
I installed jupyter-spark and I got: `jupyter-spark/extension  enabled` in the output, not jupyter_spark.  (Also not the slash vs space.)

Do you need all four?  Can you be validated but not enabled and vice versa?  (I don't know much about the extensions architecture.)

Also, looks like you'll get 4 messages if spark-jupyter isn't installed, which isn't ideal.